lib/active_storage_validations/matchers/attached_validator_matcher.rb

```diff
@@ -47,7 +51,7 @@ def matches?(subject)
       private
 
       def is_valid_when_file_attached?
-        attach_dummy_file unless file_attached?
+        attach_file(dummy_file) unless file_attached?
```

Comment:
did u try it like `attach_file unless file_attached?`?
since we have `dummy_file` as the default parameter for `file`